src/main/java/website/ui/pages/Login.java

```diff
@@ -0,0 +1,63 @@
+package website.ui.pages;
+
+import core.selenium.utils.ReaderFile;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Login class, create a PageObject of Login page
+ * to interact with the UI of that page.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class Login extends BasePage {
+
+    @FindBy(id = "username")
+    private WebElement usernameTxt;
+
+    @FindBy(id = "password")
+    private WebElement passwordTxt;
+
+    @FindBy(xpath = "//button[@id='signin-submit2' or @title='Log In']")
+    private WebElement logInBtn;
+
+    /**
+     * Writes in usernameTxt WebElement the username parameter.
+     *
+     * @param username is to write in usernameTxt WebElement.
+     */
+    public void writeInUsername(String username) {
+        usernameTxt.clear();
+        usernameTxt.sendKeys(username);
+    }
+
+    /**
+     * Writes in passwordTxt WebElement the password parameter.
+     *
+     * @param password is to write in passwordTxt WebElement.
+     */
+    public void writeInPassword(String password) {
+        passwordTxt.clear();
```

Comment:
You have similar code for writeInUsername method. Please consider to have a method that receives a WebElement and a text and clears and calls clear method and sendKeys method